### ISSUE TITLE
Xcode 11 compilation error because of plist inclusion to compilation sources

### ISSUE THREAD

**Issue by RemmoYu** [Sh1d0w/multi_image_picker#251]
**Bug Report**

`multi_image_picker` version: 

Is it happening on Android:   No 

Is it happening on iOS:  Yes 

```
 ➜  ios git:(master) ✗ flutter doctor -v                                     
[✓] Flutter (Channel master, v1.10.2-pre.30, on Mac OS X 10.15 19A546d, locale
    zh-Hans-CN)
    • Flutter version 1.10.2-pre.30 at /Users/remmo/Flutter/flutter
    • Framework revision 8504f3ae13 (3 hours ago), 2019-09-10 17:27:30 -0700
    • Engine revision 7ea9884ab0
    • Dart version 2.5.0 (build 2.5.0-dev.4.0 be66176534)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/remmo/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.0)
    • Xcode at /Applications/Xcode-beta.app/Contents/Developer
    • Xcode 11.0, Build version 11M392r
    • CocoaPods version 1.8.0.beta.2

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 39.0.3
    • Dart plugin version 191.8423
    • Java version OpenJDK Runtime Environment (build
      1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.38.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.4.1

[✓] Connected device (1 available)
    • iPhone XR • 1E6AB83C-E26B-43D0-B211-920319609200 • ios •
      com.apple.CoreSimulator.SimRuntime.iOS-13-0 (simulator)

• No issues found!

```

**Describe the bug**

flutter build ios --release
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    error: Unexpected duplicate tasks:
    1) Target 'BSGridCollectionViewLayout' (project 'Pods') has copy command from
    '/Users/remmo/Work/dyuit_shop/ios/Pods/BSGridCollectionViewLayout/Sources/InfoSource.plist' to
    '/Users/remmo/Work//dyuit_shop/build/ios/Release-iphoneos/BSGridCollectionViewLayout/BSGridCollectionViewLayo
    ut.framework/InfoSource.plist'
    2) Target 'BSGridCollectionViewLayout' (project 'Pods') has copy command from
    '/Users/remmo/Work/dyuit_shop/ios/Pods/BSGridCollectionViewLayout/Sources/InfoSource.plist' to
    '/Users/remmo/Work/dyuit_shop/build/ios/Release-iphoneos/BSGridCollectionViewLayout/BSGridCollectionViewLayo
    ut.framework/InfoSource.plist'
    note: Using new build systemnote: Planning buildnote: Constructing build description

**It should related to:  https://github.com/mikaoj/BSGridCollectionViewLayout/issues/9**

**Comment by Sh1d0w** [comment]
Thanks for the report. Opened issue in the iOS image picker library to ask for a new version with this fix https://github.com/mikaoj/BSImagePicker/issues/220 Once released I will update my plugin as well.

**Comment by Sh1d0w** [comment]
Hi, this should be fixed in version 4.5.4.
